### PR TITLE
fix: SystemAdvisor table row count is fixed

### PR DESCRIPTION
**Pull request by mkholjuraev**
Fixes incorrect row count in SystemAdvisor page:

image

### CODE REVIEW

File: src/SmartComponents/SystemAdvisor/SystemAdvisor.js [hunk 1]
```diff
@@ -64,6 +64,7 @@ const BaseSystemAdvisor = () => {
   const [inventoryReportFetchStatus, setInventoryReportFetchStatus] =
     useState('pending');
   const [rows, setRows] = useState([]);
+  const [rowCount, setRowCount] = useState(0);
```
Comment (bastilian) [inline src/SmartComponents/SystemAdvisor/SystemAdvisor.js:67]: Does it really have to be in a State?

Comment (mkholjuraev) [reply src/SmartComponents/SystemAdvisor/SystemAdvisor.js:67]: hmm.  it makes sense. Do you think useRef might be good option?

Comment (mkholjuraev) [reply src/SmartComponents/SystemAdvisor/SystemAdvisor.js:67]: if I put a simple variable inside component, the variable gets reinitialized every rerender.

Comment (bastilian) [reply src/SmartComponents/SystemAdvisor/SystemAdvisor.js:67]: You could use `useMemo` and make it depend on the `rows`, but generally I feel  just using a simple variable that doesn't require too much computing should be fine.

Comment (mkholjuraev) [reply src/SmartComponents/SystemAdvisor/SystemAdvisor.js]: I have changed to useRef.

Comment (bastilian) [reply src/SmartComponents/SystemAdvisor/SystemAdvisor.js]: The reason why i'm a bit wary about using a state for this is that it needs to be maintained, so every time the rows change it has to change another state as well.

Comment (mkholjuraev) [reply src/SmartComponents/SystemAdvisor/SystemAdvisor.js]: yeap, I have pushed new changes  to implement with useRef. I believe that useRef does not require computation as you said :).

**Review by MichalSajdik — APPROVED** [review]

**Pull request merged**